package/scripts/presto_coordinator.py

```diff
@@ -39,17 +42,22 @@ def start(self, env):
         Execute('{0} start'.format(daemon_control_script))
         if 'presto_worker_hosts' in host_info.keys():
             all_hosts = host_info['presto_worker_hosts'] + \
-                host_info['presto_coordinator_hosts']
+                        host_info['presto_coordinator_hosts']
         else:
             all_hosts = host_info['presto_coordinator_hosts']
-        smoketest_presto(
-            PrestoClient('localhost','root',
-                         config_properties['http-server.http.port']),
-            all_hosts)
+        # sleep some seconds for waiting coordinator start finished
+        time.sleep(30)
```

Comment:
@petroav As I said above,  I think I have fixed the root cause of the interruption of the error, so this code can now be removed. It may also be possible to keep this code, and the sleep time setting is shorter, just if there are other exceptions that are thrown and then interrupted? What is your opinion?